src/json/mod.rs

```diff
@@ -11,7 +11,15 @@ pub fn display_as_json(edn: &Edn) -> String {
         Edn::Symbol(s) | Edn::Str(s) => format!("{s:?}"),
         Edn::Int(n) => format!("{n}"),
         Edn::UInt(n) => format!("{n}"),
-        Edn::Double(n) => format!("{n}"),
+        Edn::Double(n) => {
+            // Rust formats an f64 with a value of 2^5 as "32".
+            // We do this to ensure all precision is printed if available, but still adds a decimal point for json.
+            let mut s = format!("{n}");
+            if !s.contains('.') {
```

Comment:
Maybe I missed it, but is there a test for this?

Reply:
Right below, line 136. Actually got a bit lucky because https://github.com/naomijub/edn-rs/blob/master/examples/edn_to_json.rs let me find the edge case. It's a double check, but I thought it should be in this file too, checking `32f64`

Reply:
Nice!